website/content/docs/configuration/credential-management/configure-credential-injection.mdx

```diff
@@ -24,6 +24,8 @@ Credential injection provides end users with a passwordless experience when they
 
 - You must have a static credential saved in your static credential store or Vault credential store. The credential must correspond to the target to which you want to authenticate.
 
+- For SSH credential injection, your SSH server must be configured to allow password authentication. Keyboard-interactive authentication is not supported.
```

Comment:
I wonder if inverting this would be more clear, e.g:

`Keyboard-interactive is not supported, when using UsernamePassword credentials ensure your SSH....`

Reply:
Yes, that sounds good to me, thanks! I inverted the phrasing and specifically called out Username password credentials.